MobPlugin/src/main/java/eu/cloudservice/cloudmob/util/ReflectionUtil.java

```diff
@@ -0,0 +1,71 @@
+package eu.cloudservice.cloudmob.util;
+
+import eu.cloudnetservice.cloudmob.ServerMob;
+import org.bukkit.Bukkit;
+import org.bukkit.Location;
+import org.bukkit.entity.ArmorStand;
+import org.bukkit.entity.EntityType;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.Wither;
+
+public final class ReflectionUtil {
```

Comment:
I don't think this would be needed anymore

Reply:
We needted for armorstand